### PR TITLE
Qr code scanner

### PR DESCRIPTION
**Pull request by shahinnezam**
All code is working and error free.  View is still needing to be set up properly and tweaked for final

### CODE REVIEW

**Review by asb43 — APPROVED** [review]
looks good

**Comment by shahinnezam** [comment]
Current pull request works with no bugs waiting on one more approval

**Review by AndrealHarrison — APPROVED** [review]
Good job

**Pull request merged**
